### ISSUE TITLE
Update to latest SVNKit 1.9.3+

### ISSUE THREAD

**Issue by dmarteinson** [subclipse/subclipse#76]
I see that the latest stable version of SVNKit is 1.9.3 as of 8 May 2018, are there any plans to update from 1.8.12 -> 1.9.x in the near future?

**Comment by lidochca** [comment]
Same question from me, 
When I try to install from latest release '4.2.4' having installed in advance 'org.tmatesoft.svnkit '1.9.3, I am getting following error:

`!MESSAGE Missing requirement: SVNKit Client Adapter 1.8.12.1 (org.tigris.subversion.clientadapter.svnkit.feature.feature.group 1.8.12.1) requires 'org.tmatesoft.svnkit [1.8.0,1.9.0)' but it could not be found`

Why you are not removing that 1.9.0 Exclusive in your next builds ? basically I had to uninstall 'org.tmatesoft.svnkit' newer version 1.9.3 and to install an older version 1.8.15, and the installation worked.

Thank you
Lidia

**Comment by markphip** [comment]
I've relaxed the version check in the snapshot build of Subclipse but when I install SVNKit 1.9.3 after that version of the library fails to load due to some error with their SQLJet library.  Only solution was to uninstall it and go back to the 1.8 version.  It could be something wrong with the way SVNKit created their Eclipse update site for this version.  I have reported problems to them in the past about this.  I have not figured it out yet though.

`java.lang.ClassNotFoundException: org.tmatesoft.svn.core.SVNKitActivator cannot be found by org.tmatesoft.sqljet_1.1.11.r1325
	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:508)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:419)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:411)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.loadBundleActivator(BundleContextImpl.java:762)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:716)
	at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:1002)
	at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:354)
	at org.eclipse.osgi.container.Module.doStart(Module.java:581)
	at org.eclipse.osgi.container.Module.start(Module.java:449)
	at org.eclipse.osgi.framework.util.SecureAction.start(SecureAction.java:468)
	at org.eclipse.osgi.internal.hooks.EclipseLazyStarter.postFindLocalClass(EclipseLazyStarter.java:114)
	at org.eclipse.osgi.internal.loader.classpath.ClasspathManager.findLocalClass(ClasspathManager.java:505)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.findLocalClass(ModuleClassLoader.java:328)
	at org.eclipse.osgi.internal.loader.BundleLoader.findLocalClass(BundleLoader.java:392)
	at org.eclipse.osgi.internal.loader.sources.SingleSourcePackage.loadClass(SingleSourcePackage.java:36)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:466)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:419)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:411)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:150)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at org.tmatesoft.svn.core.wc2.SvnOperationFactory.registerRunners(SvnOperationFactory.java:264)
	at org.tmatesoft.svn.core.wc2.SvnOperationFactory.<init>(SvnOperationFactory.java:243)
	at org.tmatesoft.svn.core.wc2.SvnOperationFactory.<init>(SvnOperationFactory.java:222)
	at org.tmatesoft.svn.core.javahl17.SVNClientImpl.<init>(SVNClientImpl.java:76)
	at org.tmatesoft.svn.core.javahl17.SVNClientImpl.<init>(SVNClientImpl.java:70)
	at org.tmatesoft.svn.core.javahl17.SVNClientImpl.newInstance(SVNClientImpl.java:43)
	at org.tigris.subversion.svnclientadapter.svnkit.SvnKitClientAdapter.<init>(SvnKitClientAdapter.java:46)
`

**Comment by markphip** [comment]
This looks like the problem to watch:  https://issues.tmatesoft.com/issue/SVNKIT-724

**Comment by markphip** [comment]
I've been in contact with SVNKit and they posted a 1.10.0 version that fixes the problems I saw.  I have updated our snapshot update site so that it pulls this in:

https://dl.bintray.com/subclipse/snapshots/

However, I have noted to them that their library at runtime is reporting a version of 1.9.2 so will need to see if a 1.10.1 needs to be created to resolve this or not.  I have not heard back from them yet.  One way or another, when we roll our next release will include the latest version.
